test/recipes/30-test_evp_data/evpciph.txt

```diff
@@ -288,7 +288,108 @@ IV = 39F23369A9D9BACFA530E26304231461
 Plaintext = F69F2445DF4F9B17AD2B417BE66C3710
 Ciphertext = B2EB05E2C39BE9FCDA6C19078C6A9D1B
 
-# We don't support CFB{1,8}-AESxxx.{En,De}crypt
+# CFB1-AES128.Encrypt
+Cipher = AES-128-CFB1
+Key = 2b7e151628aed2a6abf7158809cf4f3c
+IV = 000102030405060708090a0b0c0d0e0f
+Operation = ENCRYPT
```

Comment:
If you don't specify direction, it will exercise vector both ways.

Reply:
Ah, I did not know.
But I wanted to follow the example of the other
CFB-128 tests below.